inst/csv/treatmentPatternsRdms.csv

```diff
@@ -3,15 +3,15 @@ analyses,analysis_id,int,yes,no,Analysis identifier
 analyses,description,varchar,yes,no,Analysis description
 arguments,analysis_id,int,yes,no,Analysis identifier
 arguments,arguments,varchar,no,no,Arguments as JSON
-arguments,database_id,integer,TRUE,TRUE,Unique identifier for the database
+arguments,database_id,varchar,yes,yes,Unique identifier for the database
```

Comment:
@SivamshIndukuri thanks for making these updates - can you adjust these to be `Yes` or `No` instead of `yes` or `no`? Thanks!